hypothesis-python/docs/changes.rst

```diff
@@ -18,6 +18,23 @@ Hypothesis 6.x
 
     .. include:: ../RELEASE.rst
 
+.. _v6.34.0:
+
+-------------------
+6.34.0 - 2021-12-29
+-------------------
```

Comment:
Looks like you accidentally committed the new changelog!  Best to revert this (and the version number bump); it should be handled by our CI infra.